cypress/support/commands.ts

```diff
@@ -54,3 +54,59 @@ Cypress.Commands.add(
     cy.get('button[type="submit"]').click(); // Submit
   },
 );
+
+Cypress.Commands.add(
+  'createCourse',
+  ({
+    courseAlias,
+    showScoreboard = false,
+    startDate = new Date(),
+    unlimitedDuration = true,
+    endDate = new Date(),
+    school = 'omegaup',
+    basicInformation = false,
+    requestParticipantInformation = 'no',
+    problemLevel = 'introductory',
+    description = 'This is the description',
+    objective = 'This is an objective',
+  }) => {
```

Comment:
```suggestion
  }: CourseOptions) => {
```

así todas las variables tienen tipos! igual con los otros comandos (oops, se me fueron)

Reply:
ah aguanta, hay algunas cosas opcionales. para poder manejar los defaults y dejar como opcionales algunos de los campos, se puede cambiar a

```suggestion
  }: Partial<CourseOptions> & Pick<CourseOptions, 'courseAlias'>) => {
```

así `courseAlias` es el único parámetro requerido.

Reply:
Si tienen tipos las funciones porque se están declarando dentro de `cypress/support/cypress.d.ts` y al momento de querer usar la función si lo marca. 

```typescript
interface Chainable {
  login(loginOptions: LoginOptions): void;
  register(loginOptions: LoginOptions): void;
  createProblem(problemOptions: ProblemOptions): void;
  createCourse(
    courseOptions: Partial<CourseOptions> &
      Pick<CourseOptions, 'courseAlias'>,
  ): void;
}
```

Reply:
ah pero _esta_ función (el closure que está en la línea 60) no tiene tipos! onda, cuando alguien mande llamar `createCourse` ahí sí está declarado el tipo y va a marcar un error, pero aquí TypeScript no sabe qué debería ser `courseAlias` (porque es el único que no tiene un default).

Onda, hay que cambiar tanto `interface Chainable` como este archivo.

Reply:
ohh ya entiendo. va, gracias!